SRC/NewPortfolio/about.css

```diff
@@ -0,0 +1,122 @@
+body {
+  width: 100%;
+  height: auto;
+  /* border: 1px solid red; */
+  padding: 10%;
+  font-family: 'Roboto Mono', monospace;
+  color: #5D5C61;
+  background-color: #5D5C61;
+}
+
+.main-container {
+  width: 100%;
+  height: auto;
+  /* border: 1px solid blue; */
+  justify-content: center;
+  background-color: #b1A296;
+}
+
+/* Nav Section */
+nav {
+  background-color: #557A95;
+  height: 50px;
+  width: 100%;
+  /* border: 1px solid purple; */
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  color: ghostwhite;
+}
+
+ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.5rem;
+}
+li {
+  padding: 10px;
+}
+li:hover {
+  color: #7395AE;
+}
+
+/* End of Nav Section  */
+
+/* Start of Header */
+.image {
+  position: relative;
+  padding-top: 56.25%; /* 16:9 Aspect Ratio */
+
+}
+img {
+  position: absolute;
+  left: 0;
+  top: 0;
```

Comment:
remove position, left, top. not necessary if we remove the padding-top & add height: auto; on `.image`